components/explore/component.js

```diff
@@ -76,6 +77,16 @@ const Explore = ({
   const [interactiveFeatures, setInteractiveFeatures] = useState(null);
   const [showTour, setShowTour] = useState(false);
 
+  // User data modal, prevent modal to appear again after first visit
+  const myStorage = typeof window !== 'undefined' ? window.localStorage : null;
+  const modalStatus = typeof window !== 'undefined' && myStorage.getItem('modal');
+  const [userModalOpen, setUserModalOpen] = useState(modalStatus === null);
```

Comment:
Can you use a more descriptive name? In a few months, we won't remember what “modal” is.